### PR TITLE
JDBC Performance Tests

### PR DESCRIPTION
**Pull request by jasonkuster**
Signed-off-by: Jason Kuster <jasonkuster@google.com>

Be sure to do all of the following to help us incorporate your contribution
quickly and easily:

 - [ ] Make sure the PR title is formatted like:
   `[BEAM-<Jira issue #>] Description of pull request`
 - [ ] Make sure tests pass via `mvn clean verify`. (Even better, enable
       Travis-CI on your fork and ensure the whole test matrix passes).
 - [ ] Replace `<Jira issue #>` in the title with the actual Jira issue
       number, if there is one.
 - [ ] If this contribution is large, please file an Apache
       [Individual Contributor License Agreement](https://www.apache.org/licenses/icla.txt).

---

### CODE REVIEW

**Comment by jasonkuster** [comment]
Run Seed Job

**Comment by jasonkuster** [comment]
Run Seed Job

**Comment by jasonkuster** [comment]
Run Dataflow JDBC Performance Tests

**Comment by jasonkuster** [comment]
Run Seed Job

**Comment by jasonkuster** [comment]
Run Dataflow JDBC Performance Tests

**Comment by jasonkuster** [comment]
Run Seed Job

**Comment by jasonkuster** [comment]
Run Seed Job

**Comment by jasonkuster** [comment]
Run Dataflow JDBC Performance Tests

**Comment by jasonkuster** [comment]
This is blocked on #1751

**Comment by jasonkuster** [comment]
R: @tgroh

File: .jenkins/job_beam_PerformanceTests_JDBC.groovy [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+
+    // Run job in postcommit every 6 hours and don't trigger every push.
+    common_job_properties.setPostCommit(delegate, '0 */6 * * *', false)
+
+    def argMap = [
+      benchmarks: 'beam_integration_benchmark',
+      dpb_beam_it_module: 'sdks/java/io/jdbc',
+      dpb_it_args: '--tempRoot=gs://temp-storage-for-end-to-end-tests,--project=apache-beam-testing,--postgresServerName=10.36.0.11,--postgresUsername=postgres,--postgresDatabaseName=postgres,--postgresPassword=uuinkks,--postgresSsl=false',
+      dpb_it_class: 'org.apache.beam.sdk.io.jdbc.JdbcIOIT',
+      dpb_beam_it_profile: 'io-it'
```
Comment (tgroh) [inline .jenkins/job_beam_PerformanceTests_JDBC.groovy]: Does this profile do anything (now); will it do anything in the future?

File: .jenkins/job_beam_PerformanceTests_JDBC.groovy [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+    // Set default Beam job properties.
+    common_job_properties.setTopLevelMainJobProperties(delegate)
+
+    // Run job in postcommit every 6 hours and don't trigger every push.
+    common_job_properties.setPostCommit(delegate, '0 */6 * * *', false)
+
+    def argMap = [
+      benchmarks: 'beam_integration_benchmark',
+      dpb_beam_it_module: 'sdks/java/io/jdbc',
+      dpb_it_args: '--tempRoot=gs://temp-storage-for-end-to-end-tests,--project=apache-beam-testing,--postgresServerName=10.36.0.11,--postgresUsername=postgres,--postgresDatabaseName=postgres,--postgresPassword=uuinkks,--postgresSsl=false',
```
Comment (tgroh) [inline .jenkins/job_beam_PerformanceTests_JDBC.groovy]: This is probably worth also factoring into a map.

File: .jenkins/job_beam_PerformanceTests_JDBC.groovy (hunk 1 above)
Comment (jasonkuster) [inline .jenkins/job_beam_PerformanceTests_JDBC.groovy]: This is a profile @ssisk added for the IO ITs. It configures the failsafe execution.

**Comment by jasonkuster** [comment]
Run Seed Job

**Pull request closed without merging**